selene-lib/src/lints/roblox_incorrect_roact_usage.rs

```diff
@@ -65,14 +65,28 @@ impl Lint for IncorrectRoactUsageLint {
         }
 
         for invalid_property in visitor.invalid_properties {
-            diagnostics.push(Diagnostic::new(
-                "roblox_incorrect_roact_usage",
-                format!(
-                    "`{}` is not a property of `{}`",
-                    invalid_property.property_name, invalid_property.class_name
-                ),
-                Label::new(invalid_property.range),
-            ));
+            match invalid_property.property_name.as_str() {
+                "Name" => {
+                    diagnostics.push(Diagnostic::new(
+                        "roblox_incorrect_roact_usage",
+                        format!(
+                            "`{}` is assigned through the element's key for host components",

```

Comment:
"Host component" is a confusing internal term. This instead ought to be just "Roblox instances"